workspace.json

```diff
@@ -44,6 +44,11 @@
       "tags": ["platform:node"],
       "implicitDependencies": []
     },
+    "@fluentui/cdn": {
```

Comment:
Is this all I need to do to add a new project? When I run `yarn build` after adding this I'm getting bizarre errors from other unrelated packages.